src/vm/tieredcompilation.cpp

```diff
@@ -201,6 +201,10 @@ void TieredCompilationManager::AsyncPromoteMethodToTier1(MethodDesc* pMethodDesc
         }
     }
 
+    LOG((LF_TIEREDCOMPILATION, LL_INFO10000, "TieredCompilationManager::AsyncPromoteMethodToTier1 Method=0x%x (%s::%s), code version id=0x%x queued\n",
```

Comment:
The format character for the pMethodDesc is incorrect (%x, should be %p and probably formatted as %pM like in the preexisting log call below)